components/Hubs/HubTag.tsx

```diff
@@ -0,0 +1,48 @@
+import { toTitleCase } from "~/config/utils/string";
+import Link from "next/link";
+import { Hub } from "~/config/types/hub";
+import IconButton from "~/components/Icons/IconButton";
+import { StyleSheet, css } from "aphrodite";
+import colors from "~/config/themes/colors";
+
+const HubTag = ({ hub }: { hub: Hub }) => {
+  return (
+    <Link
+      key={`/hubs/${hub.slug ?? ""}-index`}
+      href={`/hubs/${hub.slug}`}
+      onClick={(e) => e.stopPropagation()}
```

Comment:
Do we need a stopPropagation here? Sometimes this will lead to bugs. Shouldn’t the link should take care of itself without the need for a stopPropagation?